grammar.js

```diff
@@ -542,6 +552,13 @@ module.exports = grammar({
         )
       ),
 
+    let_heredoc: ($) =>
```

Comment:
I do think that naming the rule itself `heredoc` would be confusing (since there are two types of heredocs, the name wouldn't describe which), but using an alias in `let_statement` would be a good idea I think.  I'll work on it when I get home.